src/app/services/suggest.service.ts

```diff
@@ -20,15 +19,13 @@ export class SuggestService {
 
 	// TODO: make the searchParams as configureable model rather than this approach.
 	public fetchQuery(query: string): Observable<SuggestResponse> {
-		const searchParams = new HttpParams();
-		searchParams.set('q', query);
-		searchParams.set('count', SuggestService.count);
-		searchParams.set('callback', 'JSONP_CALLBACK');
-		searchParams.set('minified', SuggestService.minified_results.toString());
-		searchParams.set('order', SuggestService.order);
-		searchParams.set('orderby', SuggestService.orderby);
-
-		const jsonpUrl = SuggestService.apiUrl.toString() + `?${searchParams.toString()}`;
+
+		const jsonpUrl = 'https://api.loklak.org/api/suggest.json' +
```

Comment:
Using HttpParams(), the results are not available for most of the time and sometimes when we get results they would be consistent for all queries. Using simple format works fine and the speed and accuracy of the results are perfect since the queries are getting updated from effect really fast and easily. Though, I am looking for alternative solution for HttpParams() in the mean time the development is working and the work of HttpParams() is just to set and extract key-values. The priority is to get proper and accurate results. Now we are getting the results for ```from:user``` very fast and appropriate.